src/provider/sequencer.ts

```diff
@@ -64,6 +55,19 @@ export type SequencerProviderOptions =
       headers?: object;
     };
 
+function isEmptyQueryObject(obj?: Record<any, any>): obj is undefined {
+  return (
+    obj === undefined ||
+    Object.keys(obj).length === 0 ||
+    (Object.keys(obj).length === 1 &&
+      Object.entries(obj).every(([k, v]) => k === 'blockIdentifier' && v === null))
+  );
+}
+
+const defaultOptions: SequencerProviderOptions = {
+  network: 'goerli-alpha-2',
```

Comment:
what do you think about moving this string, along with 'mainnet-alpha'  and  'goerli-alpha' to constants file? For example, 'goerli-alpha-2' is magic string in 3 places